### ISSUE TITLE
No support for Visual Studio 2022?

### ISSUE THREAD

**Issue by habi498** [DK22Pac/plugin-sdk#135]
Hi, i couldn't install template for visual studio 2022. Could you add support for it too. Helpful for others in future.

**Comment by nathan130200** [comment]
Up!

**Comment by GokuModsz** [comment]
Has anyone been able to use VS 2022 in Plugin-SDK??

**Comment by minotheo** [comment]
> Has anyone been able to use VS 2022 in Plugin-SDK??

U can create project in VS 19 and edit, compile with VS2022.

**Comment by GokuModsz** [comment]
but I don't understand, I mean, I would have to have both vs 2019 and vs 2022, it would be too heavy, and I only use vs 2022

**Comment by GokuModsz** [comment]
When I try to create the template in vs 2022, it says that it doesn't support it, I don't understand, or using vs 2022 how could I create the template without downloading vs 2019???

**Comment by minotheo** [comment]
> but I don't understand, I mean, I would have to have both vs 2019 and vs 2022, it would be too heavy, and I only use vs 2022

yes, if you want create project using VS2022 you need to compile pluginSDK and specify lib in projects settings and pass dependicies.

**Comment by minotheo** [comment]
Lika GTA folder, DirectxAPI and others.

**Comment by GokuModsz** [comment]
I understand, I mean plugin-sdk can be compiled?

**Comment by GokuModsz** [comment]
I don't need the VS2019 build tools??

**Comment by GokuModsz** [comment]
plugin-sdk is a dll library, or is it .lib?

**Comment by minotheo** [comment]
I do not know exactly, but when i opens a VS2019 PluginSDK project it's just asked for retarget on new version of SDK.

**Comment by minotheo** [comment]
.lib

**Comment by GokuModsz** [comment]
but i use VS2022, i want to create the template using VS2022 not VS2019

**Comment by GokuModsz** [comment]
Or will you have the .lib of plugin-sdk?

**Comment by GokuModsz** [comment]
> Hi, i couldn't install template for visual studio 2022. Could you add support for it too. Helpful for others in future.

Maybe I can help you do something in VS2022 and it worked for me but it would be necessary to use it while they create the new version of plugin-sdk for VS2022

**Comment by Megasware128** [comment]
Just get the latest version of premake and use it to generate VS2022 project files

**Comment by nathan130200** [comment]
@GokuModsz Same situaiton, we are forced to install two VS versions to develop plugins.
